examples/simple_instance/variables.tf

```diff
@@ -34,3 +34,12 @@ variable "instance_name" {
 variable "region" {
   description = "The GCP region to deploy instances into"
 }
+
+variable "client_email" {
```

Comment:
All public variables should have descriptions.

Reply:
Fixed